src/fileoverview_comment_transformer.ts

```diff
@@ -149,7 +156,8 @@ export function transformFileoverviewCommentFactory(
         for (let i = 0; i < sourceFile.statements.length; i++) {
           const stmt = sourceFile.statements[i];
           // Accept the NotEmittedStatement inserted above.
-          if (i === 0 && stmt.kind === ts.SyntaxKind.NotEmittedStatement) continue;
+          if (i === 0 && stmt.kind === ts.SyntaxKind.NotEmittedStatement)
+            continue;
```

Comment:
Not sure what happened here (formatter?) but I'd brace it if you're gonna wrap it.